testing/git-crypt/APKBUILD

```
@@ -0,0 +1,31 @@
+# Contributor: Laurent Arnoud <laurent@spkdev.net>
+# Maintainer: Laurent Arnoud <laurent@spkdev.net>
+pkgname=git-crypt
+pkgver=0.6.0
+pkgrel=0
+pkgdesc="Transparent file encryption in git"
+url="https://www.agwa.name/projects/git-crypt"
+arch="all"
+license="GPL-3.0-or-later"
+depends="git"
+makedepends="openssl-dev docbook-xml docbook-xsl"
+install=""
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/AGWA/$pkgname/archive/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make ENABLE_MAN=yes
+}
+
+check() {
+	cd "$builddir"
+}
+
+package() {
+	cd "$builddir"
+	make PREFIX="$pkgdir/usr" ENABLE_MAN=yes install
```

Looks a bit funky to me. Normally the prefix and options are given to make during the build phase and install is only given DESTDIR.

https://github.com/AGWA/git-crypt/blob/master/Makefile
```suggestion
	make DESTDIR="$pkgdir" ENABLE_MAN=yes install
```

---

It did not install properly with just your suggestion so I had to do `make PREFIX="/usr" DESTDIR="$pkgdir" ENABLE_MAN=yes install` in https://github.com/alpinelinux/aports/pull/7070/commits/720d0ec991726340a1bf9cb30999aa61590a8b3c